src/main/java/org/example/homework4/MinutesCalculator.java

```diff
@@ -0,0 +1,21 @@
+package org.example.homework4;
+
+public class MinutesCalculator {
+    public void printYearsAndDays(long minutes) {
+        if (minutes > 0) {
+            if (minutes < 1440) {
+                System.out.println(minutes + " min is less than a day");
```

Comment:
Good point, but not necessary according to the task it in case of less than a day amount of minutes, the output will be like
100 min = 0 y and 0 d